### PR TITLE
capitalize proper names

### LINKED ISSUE #15: Service Name Capitalization
What is our stance on service names? Should they be capitalized? I didn't include comments on this because I'm not sure (and I'm fairly sure I didn't capitalize them in readmes I have modified).

### PR DESCRIPTION
**Pull request by harrythehat1975**
Fixes #15

### CODE REVIEW

File: content/docs/introduction/glossary.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ designed to be a fire and forget type of message.
 ### WRP
 
 Short for [Web Routing Protocol](../../wrp/) which is the core message
-envilope used by Xmidt and it's associated services.
+envelope used by XMiDT and it's associated services.
```
Comment (harrythehat1975) [inline content/docs/introduction/glossary.md]: I think `envilope` was a typo. Changed it to `envelope`

File: content/docs/operating/caduceus.md [hunk 2]
```diff
@@ -13,10 +13,10 @@ Currently Caduceus depends on [SNS](https://aws.amazon.com/sns/getting-started/)
 
 ## Configuration
 Refer to [configuration file](https://github.com/xmidt-org/caduceus/blob/master/caduceus.yaml)
-for how to configure caduceus.
+for how to configure code Caduceus.
```
[the remark this answers is not in the document]
Comment (harrythehat1975) [reply content/docs/operating/caduceus.md]: My mistake. I'll fix that.

**Comment by harrythehat1975** [comment]
CLA page isn't leading me towards a signature page

image

**Comment by kristinaspring** [comment]
Hmmm, I'm looking into how we can fix this.  Sorry about that!

**Comment by kristinaspring** [comment]
Apparently if you wait with that page open, it eventually will load the CLA for you to sign.  I'm also opening an issue with them about this, so hopefully we can get this CLA thing figured out.

**Comment by harrythehat1975** [comment]
Ok! Should be good to go! Let me know if I need to do anything else.

~H

**Comment by britneywright** [comment]
@kristinaspring can you try closing and reopening this PR to see if that gets the license/cla out of the pending state?

**Comment by harrythehat1975** [comment]
Hi Kristina!

I thought I signed it yesterday. I tried to open up the CLA site again just now and this is what I got.

[cid:image001.png@01D57A06.B1FDA440]

**Comment by kristinaspring** [comment]
@britneywright It still seems to be stuck. 😞

**Comment by kristinaspring** [comment]
Tried to make a new PR, but it has the same issue:
https://github.com/xmidt-org/docs/pull/23

**Comment by harrythehat1975** [comment]
Can you resend me the CLA page and I’ll try it again? The one I open now keep getting an error
~H

**Comment by kristinaspring** [comment]
https://cla-assistant.io/xmidt-org/docs?pullRequest=22

Can you post the error you're getting? The image you sent yesterday isn't showing up for me.

**Comment by harrythehat1975** [comment]
This is the error that keeps coming up on the CLA page

[cid:image001.png@01D57AA9.C0B00340]

**Comment by kristinaspring** [comment]
@harrythehat1975 I can't see that image.  Instead of responding through email, can you attach it on this pull request?

**Comment by kristinaspring** [comment]
@harrythehat1975 on the issue I opened against the CLA Assistant, they said they were having performance problems but they have been resolved.  Would you mind trying again? Thanks for your patience with all of this.

**Comment by harrythehat1975** [comment]
@kristinaspring The CLA passed!

**Pull request merged**